addon/components/rdfa/rdfa-editor.ts

```diff
@@ -99,48 +114,43 @@ export default class RdfaEditor extends Component {
    *
    * @private
    */
-  get hasActiveHints() {
-    return this.hintsRegistry?.activeHints?.length > 0;
+  get hasActiveHints(): boolean {
+    return this.hintsRegistry?.activeHints
+      ? this.hintsRegistry?.activeHints?.length > 0
+      : false;
   }
 
   /**
    * @property hasSuggestedHints
    */
-  get hasSuggestedHints() {
+  get hasSuggestedHints(): boolean {
     return this.suggestedHints.length > 0;
   }
 
   /**
    * @property rootModelNode
    */
-  get rootModelNode() {
+  get rootModelNode(): ModelNode {
     return this.editor.rootModelNode;
   }
-  /**
-   * Contains extra handlers for input events on the editor.
-   *
-   * @property handlers
-   * @type Ember.A
-   *
-   * @private
-   */
-  handlers = null;
 
   /**
    * editor controller
-   *
    */
-  @tracked editor;
+  @tracked editor?: PernetRawEditor;
```

Comment:
since this is now undefined, there need to be some extra undefined checks when editor is accessed

Reply:
one option is a getter that throws when editor is null (nothing should try to run before editor is initialized, this may also alert us to some initialization bugs we were having)
if that generates too many problems, simply asserting this is non-null (`@tracked editor!: PernetRawEditor`) is also an option, though we should avoid that if possible

Reply:
such a getter should also be added in `rdfa-editor.ts`, since we have the same problem here as in `content-editable.ts`

Reply:
that won't work there though, since the editor is passed in the template before init.